src/pages/assessment-scorecard-input/assessment-scorecard-input.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import AssessmentScoreCard from '../../components/AssessmentCard/AssessmentScoreCard.jsx';
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.25.2/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected use of file extension "jsx" for "../../components/AssessmentCard/AssessmentScoreCard.jsx"